### PR TITLE
For simplified installation of python libs

### PR DESCRIPTION
**Pull request by aradove**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ joplin_api
```
Comment (foxmask) [inline requirements.txt]: you need to add beautifulsoup4 too ; as I specified all of them in setup.cfg ; to be able to just do pip install baeuda, but forgot the requirements.txt ;)

Comment (aradove) [reply requirements.txt]: All right!

I added it. So it should be okay now?

Comment (foxmask) [reply requirements.txt]: almost : it's beautifulsoup4 not bs4 ;)

Comment (foxmask) [reply requirements.txt]: see https://pypi.org/project/beautifulsoup4/

Comment (aradove) [reply requirements.txt]: Okay
pip install beautifulsoup4

Now it should be okay? :P

Comment (foxmask) [reply requirements.txt]: i'm afraid no, your requirements.txt still contains bs4 not beautifulsoup4

Comment (aradove) [reply requirements.txt]: Its a - bs4
and a + beautifulsoup4

Comment (foxmask) [reply requirements.txt]: yes it should be :)

Comment (foxmask) [reply requirements.txt]: thanks:)

**Pull request merged**